ext/vscode/package.json

```diff
@@ -9,7 +9,7 @@
     "publisher": "ms-azuretools",
     "aiKey": "0c6ae279ed8443289764825290e4f9e2-1a736e7c-1324-4338-be46-fc2a58ae4d14-7255",
     "engines": {
-        "vscode": "^1.74.0"
+        "vscode": "^1.76.0"
```

Comment:
Bumping this, because the l10n API will be available in >1.75, and because the Resources extension is soon to release with a minimum version of 1.76.